cadg-db/create_cadg_db.sql

```diff
@@ -55,12 +55,13 @@ COMMENT = '				';
 DROP TABLE IF EXISTS `cadg`.`disseminator` ;
 
 CREATE TABLE IF NOT EXISTS `cadg`.`disseminator` (
```

Comment:
Based on our conversations with Reza, I was under the impression that every disseminator should have an active IP and a standby IP, along with the CMAC version being used for each. Is this being considered in this db design?

Reply:
I am going to wait until I know for sure everything else that we need in the db schema before I make further modifications. I have some questions. Like, does this mean we need 2 ports for each IP, or can we assume they each use the same 2 ports?